tests/UI/commonTests/FO/hummingbird.ts

```diff
@@ -0,0 +1,139 @@
+import loginCommon from '@commonTests/BO/loginBO';
+
+import dashboardPage from '@pages/BO/dashboard';
+import themeAndLogoPage from '@pages/BO/design/themeAndLogo/themeAndLogo';
+import themeImportPage from '@pages/BO/design/themeAndLogo/themeAndLogo/import';
+
+import helper from '@utils/helpers';
+import testContext from '@utils/testContext';
+
+import {expect} from 'chai';
+import type {BrowserContext, Page} from 'playwright';
+
+function installHummingbird(baseContext: string = 'commonTests-installHummingbird'): void {
+  describe('Install Hummingbird theme', async () => {
+    let browserContext: BrowserContext;
+    let page: Page;
+
+    const urlTheme: string = 'https://github.com/PrestaShop/hummingbird/releases/download/v0.1.5/hummingbird.zip';
```

Comment:
```suggestion
    const urlTheme: string = 'https://github.com/PrestaShop/hummingbird/releases/latest/download/hummingbird.zip';
```
To avoid changing version after each release

Reply:
@SharakPL I disagree. I will update it after each release for avoiding the nightly breaks for error in tests (and not in Hummingbird).